cvat/apps/documentation/faq.md

```diff
@@ -100,3 +102,9 @@ Annotations are stored in the PostgreSQL database. The database files are stored
 volumes:
   - cvat_db:/var/lib/postgresql/data
 ```
+
+## How to mark job/task as completed
+The status is set by the user in the [Info window](user_guide.md#info) of the job annotation view. There are three types of status: annotation, validation or completed. The status of the job changes the progress bar of the task.
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 120 characters](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5848096)